helm/nessie/templates/deployment.yaml

```diff
@@ -222,11 +222,14 @@ spec:
             - name: nessie-server
               containerPort: 19120
               protocol: TCP
+            - name: nessie-mgmt
```

Comment:
FYI this will have an impact on the K8s Operator as well, I will take care of that once this PR is merged.